kubehub/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+from . views import kubespray_deploy
+
+
+urlpatterns = [
+    url(r'^run/ansible$', kubespray_deploy.kubespray_deploy, name='run_ansible_script'),
```

Comment:
don't need to expose /ansible from public, finally we should have api lile /cluster/create /cluster/destroy etc

Reply:
Fixed 61039ed